File: FoodieApplication/frontend/FoodieApplication/src/app/favService/domain/dish.ts
import { Image } from "./image";

export class Dish {
     
    dishId:string
    dishName:string
    dishType:string;
    image:Image[];

    constructor(){
        this.dishId='';
        this.dishName='';
        this.dishType='';
        this.image=[];
    }
}

File: FoodieApplication/FavouriteService/src/main/java/com/example/FavouriteService/model/Dish.java
package com.example.FavouriteService.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Dish {

    private String dishId;
    private String dishName;
    private String dishType;
//    private Image image;
}
